new/server-enforced-recipe.md

```diff
@@ -0,0 +1,104 @@
+---
+RFC: unassigned
+Title: Server Enforced Recipe
+Author: Daniel DeLeo <dan@chef.io>
+Status: Draft
+Type: Standards Track
+<Tracking:>
+<  - https://github.com/chef/chef/issues/X>
+---
+
+# Server Enforced Recipe
+
+## Description
+
+Chef Server will provide an endpoint that MAY serve a Chef recipe file. Chef
+Client will attempt to fetch the recipe during run context setup. If
+no user action is taken to configure the feature, the endpoint returns 404
+and Client behavior will be unaffected. When the feature is enabled, the
+endpoint returns the configured recipe file. Chef Client will evaluate and
+converge the recipe.
+
+## Rationale
+
+The motivation for this feature is to allow the operator of the Chef Server to
+enforce desired client-side configuration using Chef. Intended use cases include:
+
+* Allow cloud-based vendors to install an agent necessary for correct operation
+  of the service
+* Allow Chef Customer Development Partners to efficiently install experimental
+  client-side software during feature development
+* Allow organizations that operate as internal service providers to enforce
+  standard configurations
+
+Currently the only way to achieve these goals with Chef is to enforce the
+inclusion of recipes on a node's run list, which is not feasible in many cases.
+
+The enforced policy is limited to a single recipe instead of a full cookbook or
```

Comment:
Lack of attributes is a feature in this case, since that way you're less likely to make attributes that can conflict with the attributes from the cookbooks in the run list.